#VRML_SIM R2020a utf8
WorldInfo {
}
Viewpoint {
  orientation 0.7937022719950197 0.6073200772751931 0.03462697168966976 5.835509674424804
  position -0.42481716270195463 0.764693408982795 2.579844750624047
}
TexturedBackground {
}
TexturedBackgroundLight {
}
Robot {
  translation -2.21999 -0.149998 7.08
  rotation 0.5773509358554485 0.5773489358556708 0.5773509358554485 2.0944
  children [
    HingeJoint {
      device [
        RotationalMotor {
          name "rotational motor 3"
          acceleration 1.5655595373012297
        }
      ]
      endPoint Solid {
        translation 0 0.64 0
        rotation 1 0 0 0
        children [
          Shape {
            geometry Cylinder {
              height 0.2
              radius 0.05
            }
          }
        ]
        name "solid(2)"
      }
    }
    HingeJoint {
      device [
        RotationalMotor {
          name "rotational motor 2"
          acceleration 0.72241341228803
        }
      ]
      endPoint Solid {
        translation 0 0.43 0
        rotation 1 0 0 0
        children [
          Shape {
            geometry Cylinder {
              height 0.2
              radius 0.05
            }
          }
        ]
        name "solid(1)"
      }
    }
    Shape {
      geometry Cylinder {
        height 0.2
        radius 0.05
      }
    }
    HingeJoint {
      device [
        RotationalMotor {
          name "rotational motor 1"
          acceleration 1.6086232168900203
        }
      ]
      endPoint Solid {
        translation 0 0.22 0
        rotation 1 0 0 0
        children [
          Shape {
            geometry Cylinder {
              height 0.2
              radius 0.05
            }
          }
        ]
      }
    }
  ]
  name "new robot"
  boundingObject Cylinder {
    height 0.2
    radius 0.05
  }
  physics Physics {
  }
  controller "my_controller"
  supervisor TRUE
}
Windmill {
  translation -26.17 -0.54 20.25
}
Robot {
  translation 0 0.01 0
  rotation 0.5773509358554485 0.5773489358556708 0.5773509358554485 2.0944
  children [
    HingeJoint {
      device [
        RotationalMotor {
          name "rotational motor 1"
          acceleration 1.6086232168900203
        }
      ]
      endPoint Solid {
        translation 0 0.10598100000000003 0
        rotation 1 0 0 0
        children [
          HingeJoint {
            device [
              RotationalMotor {
                name "rotational motor 2"
                acceleration 0.72241341228803
              }
            ]
            endPoint Solid {
              translation 0 0.11 0
              rotation 1 0 0 0
              children [
                HingeJoint {
                  device [
                    RotationalMotor {
                      name "rotational motor 3"
                      acceleration 1.5655595373012297
                    }
                  ]
                  endPoint Solid {
                    translation 0 0.10999999999999999 0
                    rotation 1 0 0 0
                    children [
                      Shape {
                        geometry Cylinder {
                          height 0.1
                          radius 0.05
                        }
                      }
                    ]
                    boundingObject Cylinder {
                      height 0.1
                      radius 0.05
                    }
                    physics Physics {
                    }
                  }
                }
                Shape {
                  geometry Cylinder {
                    height 0.1
                    radius 0.05
                  }
                }
              ]
              boundingObject Cylinder {
                height 0.1
                radius 0.05
              }
              physics Physics {
              }
            }
          }
          Shape {
            geometry Cylinder {
              height 0.1
              radius 0.05
            }
          }
        ]
        boundingObject Cylinder {
          height 0.1
          radius 0.05
        }
        physics Physics {
        }
      }
    }
    Shape {
      geometry Cylinder {
        height 0.1
        radius 0.05
      }
    }
  ]
  boundingObject Cylinder {
    height 0.1
    radius 0.05
  }
  physics Physics {
  }
  controller "my_controller_2"
  supervisor TRUE
}
WoodenBox {
  translation 0 -0.57 0.16
  size 10 1 10
}
